### DISCUSSION TITLE
Android Manifest configuration

### DISCUSSION THREAD

**Discussion by asgairola-doubtnut** [Sunbird-Ed/Community#22] [Sunbird ED configuration discussion] [answered]
<activity
            android:name="com.doubtnutapp.diksha.DikshaLandingActivity"
            android:label="@string/doubtnut" >
            <intent-filter android:label="@string/doubtnut" >
                <action android:name="android.intent.action.VIEW" />

                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />
                
                <data
                    android:host="sunbird.staginged.in"
                    android:pathPrefix="/play"
                    android:scheme="https" />
            </intent-filter>
            <intent-filter android:label="@string/doubtnut" >
                <action android:name="android.intent.action.VIEW" />

                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />
                
                <data
                    android:host="play"
                    android:scheme="example" />
            </intent-filter>
        </activity>

@sharathkashyap we are not able to launch the intent for the android app as not error is displayed on the console for sunbird too. 

Can you review this android side Manifest changes for intent filter... and let me know if we need to change anything here !

**Comment by sharathkashyap** [comment, marked as answer]
@swayangjit Kindly help them with Manifest entries.
cc: @luckynvdu

**Reply by swayangjit** [reply]
@asgairola-doubtnut  What is the url you are clicking to launch the intent. According to the manifest that you have posted, the URL should be `example://play?data... `.

**Reply by asgairola-doubtnut** [reply]
@swayangjit we are firing the intent from sunbird staging app .. under open with opetion on content-details page.. this was done with the help of @sharathkashyap ..


please let me know if I need to share something !

**Reply by asgairola-doubtnut** [reply]
@sharathkashyap can you help here with what @swayangjit is asking !

**Reply by swayangjit** [reply]
@asgairola-doubtnut Ok got it.

**Reply by asgairola-doubtnut** [reply]
@swayangjit this is resolved as we are able to get the data alongwith the intent !
